build.gradle

```diff
@@ -40,6 +40,10 @@ allprojects {
         def bintrayUser = findProperty('bintray.user')
         def bintrayKey = findProperty('bintray.key')
 
+        // Skip the task if this is a snapshot.
```

Comment:
We can have this but we don't need it either, the task will just fail if it's snapshot.